cfe.py

```diff
@@ -3,6 +3,65 @@
 import pandas as pd
 import sys
 
+from scipy.integrate import odeint
+from numba import jit
+import math
+
+@jit(nopython=True)
+def conceptual_reservoir_flux_calc(t, S, storage_threshold_primary_m, storage_max_m, coeff_primary, coeff_secondary, PET, infilt, wltsmc):
```

Comment:
Why not include this in the CFE class? Then all these inputs will be in CFE_STATE.

Reply:
I simply didn't try! They can be included in the CFE class